### PR TITLE
Gdrive doc

### PR DESCRIPTION
**Pull request by zweedao**

### CODE REVIEW

**Comment by zweedao** [comment]
Too much text. No need to review alr just approve @graceliching 👌

File: connectors/google-drive/action-download-file.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Google Drive action - Download file
+This action downloads contents of a file in Google Drive. File contents can be used in subsequent steps of the recipe to upload the file into other applications.
+
+Only binary files can be downloaded. Trying to download a folder or a Google Docs document will result in an error.
+
+## Input fields
+
+| Field name | Description |
+|---|---|
+| File ID | The ID of the file. Can be found in the output of other Google Drive trigger/actions, or from the file sharable link. To get sharable link, right click on the file and select `Get shareable link`. For example, if the sharable link is https://drive.google.com/file/d/1y1nxL1248oiyR2IjW_Rq84d3qEPYRoqh/view?usp=sharing , then file ID will be `1y1nxL1248oiyR2IjW_Rq84d3qEPYRoqh`.|
```
Comment (graceliching) [inline connectors/google-drive/action-download-file.md]: | File ID | The ID of the file. Can be found in the output of other Google Drive trigger/actions, or from the file's shareable link. To get the shareable link, right click on the file and select `Get shareable link`. For example, if the shareable link is https://drive.google.com/file/d/1y1nxL1248oiyR2IjW_Rq84d3qEPYRoqh/view?usp=sharing , then the file ID is `1y1nxL1248oiyR2IjW_Rq84d3qEPYRoqh`.|

File: connectors/google-drive/action-download-file.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Output fields
+
+| Field name | Description |
+|---|---|
+| File contents | The text contents of the file. Can be used in `Upload file` action of other file connectors such as Box, Dropbox, Amazon S3, etc., to upload the file into those applications.  |
```
Comment (graceliching) [inline connectors/google-drive/action-download-file.md]: | File contents | The text contents of the file. Can be used in `Upload file` actions of other file connectors such as Box, Dropbox, Amazon S3, etc. to upload the file into those applications.  |

File: connectors/google-drive/action-rename-file.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+title: Workato connectors - Google Drive trigger - New file or folder
+date: 2018-12-20 06:00:00 Z
+---
+
+# Google Drive action - Rename/move file or folder
+This actions rename or move a file or folder in Google Drive.
```
Comment (graceliching) [inline connectors/google-drive/action-rename-file.md]: This action renames or moves a file/folder in Google Drive.

File: connectors/google-drive/action-rename-file.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+## Input fields
+
+| Field name | Description |
+|---|---|
+| File or folder ID | Can be found in the output of other Google Drive trigger/actions.<br><br>Alternatively, click on the required folder and folder ID can be found at the end of URL. For file ID, right click on the file and select `Get shareable link`. For example, if the sharable link is https://drive.google.com/file/d/1y1nxL1248oiyR2IjW_Rq84d3qEPYRoqh/view?usp=sharing , then file ID will be `1y1nxL1248oiyR2IjW_Rq84d3qEPYRoqh`. |
```
Comment (graceliching) [inline connectors/google-drive/action-rename-file.md]: | File or folder ID | Can be found in the output of other Google Drive trigger/actions.<br><br>Alternatively, click on the required folder and the folder ID can be found at the end of URL. For file ID, right click on the file and select `Get shareable link`. If the sharable link is https://drive.google.com/file/d/1y1nxL1248oiyR2IjW_Rq84d3qEPYRoqh/view?usp=sharing , then the file ID is `1y1nxL1248oiyR2IjW_Rq84d3qEPYRoqh`. |

File: connectors/google-drive/action-search-files.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
+---
+title: Workato connectors - Google Drive trigger - New file or folder
+date: 2018-12-20 06:00:00 Z
+---
+
+# Google Drive action - Search files or folders
+This action retrieve a list of files or folders that matches your search criteria.
```
Comment (graceliching) [inline connectors/google-drive/action-search-files.md]: This action retrieves a list of files or folders that match your search criteria.

File: connectors/google-drive/action-search-files.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+## Input fields
+
+| Field name | Description |
+|---|---|
+| Files or folders | Select whether to search for files or for folders. |
+| Folder | Narrows down the search to within this folder. Matching files/folders in sub-folders will not be fetched. Leave blank to search through My Drive. |
+| Name | Search for files or folders whose names partially matches this given name. |
```
Comment (graceliching) [inline connectors/google-drive/action-search-files.md]: | Name | Search for files or folders whose names partially match the given name. |

File: connectors/google-drive/action-search-files.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+| -- Email address | The email address of the user. This may not be present in certain contexts if the user has not made their email address visible to the requester. |
+| -- Permission ID | The user's ID as visible in Permission resources. |
+| -- Photo link | A link to the user's profile photo, if available. |
+| -- Me | Whether this user is the requesting user. |
+| - Owners | The list of owners of the file. Currently, only certain legacy files may have more than one owner. Not populated for Team Drive files. |
+| -- Display name | A plain text displayable name for this user. |
+| -- Email address | A link to the user's profile photo, if available. |
+| -- Permission ID | The user's ID as visible in Permission resources. |
+| -- Photo link | A link to the user's profile photo, if available. |
+| -- Me | Whether this user is the requesting user. |
```
Comment (graceliching) [inline connectors/google-drive/action-search-files.md]: is this a T/F?

File: connectors/google-drive/action-search-files.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+| -- List size | Number of owners in this list. |
+| Last modifying user | The last user to modify the file. |
+| -- Display name | A plain text displayable name for this user. |
+| -- Email address | The email address of the user. This may not be present in certain contexts if the user has not made their email address visible to the requester. |
+| -- Permission ID | The user's ID as visible in Permission resources. |
+| -- Photo link | A link to the user's profile photo, if available. |
+| -- Me | 	Whether this user is the requesting user. |
+| - Shared | Whether the file has been shared. Not populated for Team Drive files. |
+| - Owned by me | Whether the user owns the file. Not populated for Team Drive files. |
+| - Viewers can copy content | Deprecated. |
```
Comment (graceliching) [inline connectors/google-drive/action-search-files.md]: If deprecated, should it still be here?

File: connectors/google-drive/action-search-files.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+| - Writers can share | Whether users with only writer permission can modify the file's permissions. Not populated for Team Drive files. |
```
Comment (graceliching) [inline connectors/google-drive/action-search-files.md]: | - Whether users with only `writer` permissions can modify the file's permissions. Not populated for Team Drive files. |

File: connectors/google-drive/action-upload-file.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+## Input fields
+
+| Field name | Description |
+|---|---|
+| File contents | Binary contents of the file to upload. Accept a data pill such as `File contents` from other Workato triggers and actions. |
+| Mime type | Mime type of this file, as stated in [Google documentation](https://developers.google.com/drive/api/v3/mime-types). |
+| File name | Name of the uploaded file. |
+| Description | Description of the file. |
+| Parent folder | The parent folder where the file will reside. Select a folder from the picklist or follow the hint to enter folder ID directly. |
+| Starred | Whether you want to starred the file. |
```
Comment (graceliching) [inline connectors/google-drive/action-upload-file.md]: | Starred | Whether you want to star the file. |

File: connectors/google-drive/action-upload-file.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+| Viewers can copy content | Deprecated. |
+| Writers can share | Whether users with only `writer` or `edit` permission can modify the file's permissions. Not populated for Team Drive files. |
```
Comment (graceliching) [inline connectors/google-drive/action-upload-file.md]: | Writers can share | Whether users with only `writer` or `edit` permissions can modify the file's permissions. Not populated for Team Drive files. |

File: connectors/google-drive/trigger-new-file.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
+---
+title: Workato connectors - Google Drive trigger - New file or folder
+date: 2018-12-20 06:00:00 Z
+---
+
+# Google Drive trigger - New file or folder
+Triggers immediately in real-time when new file or folder is created in specified folder. Sub-folders will not be monitored.
```
Comment (graceliching) [inline connectors/google-drive/trigger-new-file.md]: Triggers immediately in real-time when a new file or folder is created in a specified folder. Sub-folders will not be monitored.

File: connectors/google-drive/trigger-new-file.md [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+
+## Input fields
+
+| Field name | Description |
+|---|---|
+| Folder | The folder to monitor for new files or folders. Sub-folders will not be monitored. Select a folder from the picklist or follow the hint to enter folder ID directly. |
```
Comment (graceliching) [inline connectors/google-drive/trigger-new-file.md]: | Folder | The folder to monitor for new files or folders. Sub-folders will not be monitored. Select a folder from the picklist or enter the folder ID directly. |

**Comment by zweedao** [comment]
done @graceliching

**Pull request merged**